phpseclib/Net/SSH2.php

```diff
@@ -1347,8 +1347,8 @@ private function connect()
                     $read = [$this->fsock];
                     $write = $except = null;
                     $start = microtime(true);
-                    $sec = floor($this->curTimeout);
-                    $usec = 1000000 * ($this->curTimeout - $sec);
+                    $sec = (int)floor($this->curTimeout);
```

Comment:
`floor()` is not needed because casting to an int will already round down.
Same with the other change below.

Reply:
hi

![image](https://user-images.githubusercontent.com/870716/151477440-4d7b25f1-6c51-45a2-8c09-b6e56aa7dbce.png)

floor will response double, not int. 
of course, value is roundowned (as int).